src/SonarScanner.MSBuild.PreProcessor/WebServer/SonarQubeWebServer.cs

```diff
@@ -109,6 +109,7 @@ public override async Task<IList<SensorCacheEntry>> DownloadCache(ProcessedArgs
             catch (Exception e)
             {
                 logger.LogWarning(Resources.WARN_IncrementalPRCacheEntryRetrieval_Error, e.Message);
+                logger.LogDebug(e.ToString());
```

Comment:
Do you remember why SonarQube behaves [different than SonarCloud](https://github.com/SonarSource/sonar-scanner-msbuild/blob/master/src/SonarScanner.MSBuild.PreProcessor/WebServer/SonarCloudWebServer.cs#L100) here?

What I mean is, why the log-severity is different? 

I suggest to normalize it to be consistent (Tests will not be that painful, you should only have 1 failing I think.)

Reply:
They just got out of sync. I'm changing the other one too.